include/drivers/lora.h

```diff
@@ -139,4 +149,25 @@ static inline int lora_recv(struct device *dev, u8_t *data, u8_t size,
 	return api->recv(dev, data, size, timeout, rssi, snr);
 }
 
+/**
+ * @brief Transmit a continuous wave at a given frequency
```

Comment:
Need to mention `unmodulated` here. Also, it'd be good to mention the use of this API.